pcsx2/IopMem.cpp

```diff
@@ -490,3 +490,14 @@ void __fastcall iopMemWrite32(u32 mem, u32 value)
 		}
 	}
 }
+
+std::string iopMemReadString(u32 mem, int maxlen)
```

Comment:
I really don't see why we should be concerned about the performance of this at all, it's only going to be invoked a few times per boot during recompilation and when logging.  An absolute guarantee of safety and correctness seems more important.